continuous_integration/gpuci/environment-3.8.yaml

```diff
@@ -0,0 +1,49 @@
+name: dask-sql
+channels:
+- rapidsai
+- rapidsai-nightly
+- nvidia
+- conda-forge
+- nodefaults
+dependencies:
+- dask>=2022.3.0
```

Comment:
Is this file checking for min dependencies except dask/pyarrow/numpy since that conflicts with the latest rapids version?

Reply:
Yup, this is essentially the 3.8 environment with the following modifications:

https://github.com/rapidsai/dask-build-environment/blob/c4aac1a7718413965d382b4e9f92f9d1028561af/dask_sql/Dockerfile#L35-L38

Merged with some RAPIDS requirements